app/Jobs/Middleware/GambitRateLimit.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Rogue\Jobs\Middleware;
+
+use Illuminate\Support\Facades\Redis;
+
+class GambitRateLimit
+{
+    /**
+     * Process the queued job.
+     *
+     * @param  mixed  $job
+     * @param  callable  $next
+     * @return mixed
+     */
+    public function handle($job, $next)
+    {
+        // Rate limit to 10 requests/second.
+        $throttler = Redis::throttle('gambit')
```

Comment:
I've created a second rate limiter (in addition to `CustomerIoRateLimit`) since this is a separate service and therefore shouldn't count towards the same limit as Customer.io. I figured 10 requests/second was a decent max, but happy to adjust to whatever makes the most sense.

cc: @aaronschachter